### PR TITLE
 Add multiline-field

### PR DESCRIPTION
**Pull request by edetec**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "build": "npm run build:lib && npm run build:demo",
     "build:lib": "ng-packagr -p lib/package.json",
     "build:demo": "ng build -prod",
-    "ci_test": "./node_modules/.bin/ng test --single-run",
+    "ci_test": "ng test --single-run",
```
Comment (igortg) [inline package.json]: Are you sure `ng` is available on path?

File: lib/dynamic-form/fields/multiline-field.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { DynamicField } from './dynamic-field';
+
+export class MultilineField extends DynamicField {
```
Comment (igortg) [inline lib/dynamic-form/fields/multiline-field.ts]: Some docs

File: lib/dynamic-form/fields/multiline-field.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  public controlType ? = 'MULTILINE';
+  public rows?: number;
```
Comment (igortg) [inline lib/dynamic-form/fields/multiline-field.ts]: Doc here? What is `rows`?

File: package.json (hunk 1 above)
Comment (edetec) [reply package.json]: yes

**Pull request merged**